fusionauth/resource_fusionauth_tenant_test.go

```diff
@@ -546,6 +547,7 @@ resource "fusionauth_tenant" "test_%[1]s" {
     reset_count_in_seconds = 600
     too_many_attempts      = 3
     action_cancel_policy_on_password_reset = true
+	email_user			   = true
```

Comment:
This indentation doesn't look correct, but it could be an issue with the PR viewer.

Reply:
Sorry about that, should be fixed in https://github.com/gpsinsight/terraform-provider-fusionauth/pull/226/commits/1c90878db91bf8fd8a5d4740929fa6ebd8f20dca

Reply:
And fixed some more in https://github.com/gpsinsight/terraform-provider-fusionauth/pull/226/commits/380b432730f942f9cc557cf58eca56fb9576200d